project/src/main/world/creature/creature-visuals.gd

```diff
@@ -560,9 +555,12 @@ func _update_creature_properties() -> void:
 			if has_node(node_path):
 				get_node(node_path).material.set_shader_param(shader_param, shader_value)
 	
-	if has_node("Body/Viewport/Body"):
-		$Body/Viewport/Body.update()
+	scale = Vector2(0.60, 0.60) if dna.get("body") == "2" else Vector2(1.00, 1.00)
 	visible = true
+	
+	# reset the mouth/eye frame, otherwise we have one strange transition frame
```

Comment:
This also initializes the creature curves; this is worth mentioning in the comment

Reply:
Rephrased to `initialize creature curves, and reset the mouth/eye frame...`